MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -1221,6 +1221,7 @@ std::string EnggDiffractionViewQtGUI::fittingRunNo() const {
 }
 
 std::string EnggDiffractionViewQtGUI::fittingPeaksData() const {
+  // this should be moved to Helper or could use the poco string tokenizers
```

Comment:
Ops, we just have to remember to avoid the Poco tokenizers and use the Kernel::StringTokenizer instead if we end up using tokenizers. There were issues with the Poco library, so better to stay away from it.